packages/terra-core-docs/src/terra-dev-site/doc/pills/Pills.1.doc.mdx

```diff
@@ -0,0 +1,74 @@
+import { Badge } from 'terra-pills/package.json?dev-site-package';
+import { Notice } from '@cerner/terra-docs';
+
+import BasicPillExample from './example/BasicPill?dev-site-example';
+import BasicRemovablePillExample from './example/BasicRemovablePill?dev-site-example';
+import DisclosurePillExample from './example/DisclosurePill?dev-site-example';
+import DisclosureRemovablePillExample from './example/DisclosureRemovablePill?dev-site-example';
+
+import PillListPropsTable from 'terra-pills/src/PillList?dev-site-props-table';
+import PillsPropsTable from 'terra-pills/src/Pill?dev-site-props-table';
+
+<Badge />
+
+# Terra Pills
+
+Pills are used to assign a label or keyword to user interface (UI) objects, defining a non-hierarchal group. They are used to visually label UI objects for quick recognition. These labels can be generated either by the system or created by users entering data.
+
+## What the component is
+
+Pills are comprised of the following:
+* Pill container with actionable target area
+* Label
+
+There are two types of pills:
+* Basic
+* Removable
+
+## What the component is not
+
+* Although similar, pills and tags have their own use cases and best practices for when they should be used within the UI. See tags for more information.
```

Comment:
```suggestion
* Although similar, pills and tags have their own use cases and best practices for when they should be used within the UI. See [tags](https://engineering.cerner.com/terra-ui/components/terra-tag/tag/tag) for more information.
```

Reply:
This should work too and will keep you on the current site you're on, terra-core or terra-ui.
```suggestion
Although similar, pills and tags have their own use cases and best practices for when they should be used within the UI. See [tags](/components/terra-tag/tag/tag) for more information.
```